src/pages/docs/infrastructure/deployment-targets/kubernetes/index.md

```diff
@@ -0,0 +1,28 @@
+| Security                                             | No need to configure firewall<br />No need to provide external access to cluster                                                                                | Depends on the cluster configuration                                                                |
+| Requires workers                                     | No                                                                                                                                                              | Yes                                                                                                 |
+| Requires public IP                                   | No                                                                                                                                                              | Yes                                                                                                 |
+| Requires service account in Octopus                  | No                                                                                                                                                              | Yes                                                                                                 |
+| Limit deployments to a namespace                     | Yes                                                                                                                                                             | No                                                                                                  |
+| Planned support for upcoming observability  features | Yes                                                                                                                                                             | No                                                                                                  |
+| Recommended usage scenario                           | <ul><li>For deployments and maintenance tasks (runbooks) on Kubernetes</li><li>If you want to run a worker on Kubernetes (to deploy to other targets)</li></ul> | If you cannot install an agent on a cluster                                                         |
+| Step configuration                                   | Simple (you need to specify target tag)                                                                                                                         | More complex (requires target tags, workers, execution container images)                            |
+| Maintenance                                          | <ul><li>Octopus will update the agent</li><li>No need to add and manage</li></ul> credentials                                                                   | <ul><li>You need to update/rotate credentials</li><li>Requires worker maintenance updates</li></ul> |
```

Comment:
`Octopus will update the agent` -> `Upgradeable via Octopus Server`?

atm it sounds like it's done automagically

Reply:
updated